documents/coredocument.go

```diff
@@ -55,22 +55,23 @@ const (
 	SignaturesTreePrefix = "signatures_tree"
 )
 
+var compactProps = map[string][]byte{
```

Comment:
I can see that you are following some convention by extracting the map out of the function.